boards/x86/up_squared/Kconfig.board

```diff
@@ -2,5 +2,10 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config BOARD_UP_SQUARED
-	bool "UP Squared"
+	bool "UP Squared (x86)"
 	depends on SOC_APOLLO_LAKE
+
+config BOARD_UP_SQUARED_64
```

Comment:
I would make this board 64bit only, I do not think we should support 32bit by default.

Reply:
I think it would be good to eventually have two build targets, one 64bit and one for 32-bit HW testing since we have these on hand

Reply:
Split the difference and make the default "up_squared" board a 64 bit platform and have an "up_squared_32" for oddball uses?  I mostly agree with Anas that real applications on 64 bit CPUs are very rarely going to actually want to build a 32 bit target.

Reply:
I was about to suggest the same (i.e. up_squared_32)

Reply:
this sounds good to me